app/src/main/java/com/kickstarter/libs/KSCurrency.java

```diff
@@ -96,7 +76,8 @@ public String formatWithProjectCurrency(final double initialValue, final @NonNul
    */
   public String formatWithUserPreference(final double initialValue, final @NonNull Project project, final @NonNull RoundingMode roundingMode) {
 
-    final CurrencyOptions currencyOptions = userCurrencyOptions(initialValue, roundingMode, project);
+    final float unconvertedValue = roundingMode == RoundingMode.UP ? (float) Math.ceil(initialValue) : (float) Math.floor(initialValue);
```

Comment:
To avoid repetition here, could you create a helper function that returns the rounded value? Something like
```
private float roundedValue(final @NonNull double initialValue, final @NonNull RoundingMode roundingMode) {
   return roundingMode == RoundingMode.UP ? (float) Math.ceil(initialValue) : (float) Math.floor(initialValue);
}
```
So you simply use:
```
final float unconvertedValue = this.roundedValue(initialValue, roundingMode);
```
(Forgive my Java skills 😛 )